application/github/v0/config/tasks.json

```diff
@@ -0,0 +1,1607 @@
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 10,
+          "type": "string"
+        },
+        "sort": {
+          "default": "created",
+          "title": "Sort",
+          "description": "Sort the issues by created, updated, popularity, or long-running. Default is created",
+          "enum": [
+            "created",
+            "updated",
+            "popularity",
+            "long-running"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 11,
+          "type": "string"
+        },
+        "direction": {
+          "default": "desc",
+          "title": "Direction",
+          "description": "Direction of the sort, can be one of: asc, desc. Default is asc",
+          "enum": [
+            "asc",
+            "desc"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 12,
+          "type": "string"
+        },
+        "since": {
+          "default": "2021-01-01T00:00:00Z",
+          "title": "Since",
+          "description": "Only issues updated at or after this time are returned. Default is 2021-01-01T00:00:00Z",
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 13,
+          "type": "string"
+        },
+        "no_pull_request": {
+          "title": "No Pull Request",
+          "description": "Whether to not include pull requests in the issues. Default is false",
+          "instillFormat": "boolean",
+          "instillAcceptFormats": [
+            "boolean"
+          ],
+          "instillUpstreamTypes": [
+            "value"
+          ],
+          "instillUIOrder": 14,
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "owner",
+        "repository"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "All issues in GitHub repository",
+      "instillUIOrder": 0,
+      "properties": {
+        "issues": {
+          "description": "An array of issues",
+          "instillUIOrder": 1,
+          "title": "Issues",
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/issue",
+            "required": [],
+            "description": "An issue in GitHub"
+          }
+        }
+      },
+      "required": [
+        "issues"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_GET_ISSUE": {
```

Comment:
Same question above.
If we remain multiple one with params `issue_number`, will it be cleaner in terms of UX / codebase?